atm.js

```diff
@@ -1,21 +1,64 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+$(document).ready (function(){
```

Comment:
the document.ready here works, but is unecessary. Because this file is loaded just before the closing body tag, the DOM will have already loaded